ci/docker/debug-cpu.yaml

```
@@ -18,15 +18,22 @@ spack:
       true
 
   specs:
-    - dla-future@master build_type=Debug +miniapps +ci-test +ci-check-threads ^openblas threads=openmp ^mpich
+    - dla-future@master build_type=Debug +miniapps +ci-test +ci-check-threads
 
   packages:
     all:
       variants:
         - 'build_type=Release'
+    blas:
+      require:: 'openblas'
+    lapack:
+      require:: 'openblas'
```

Related to comment above for `intel-mkl`.
```suggestion
    blas:
      require: 'openblas threads=openmp'
    lapack:
      require: 'openblas threads=openmp'
```
?